tests/test_auditlog.py

```diff
@@ -62,3 +69,30 @@ def test_author_log_delete(self):
         self.assertEqual(auditlog.action, 'DELETE')
         # auditlog.payload.pop('name')
         self.assertEqual(auditlog.payload['data']['given_name'], self.author_detached['given_name'])
+
+    def test_audit_log_contribution(self):
+        models.AuditLog.objects.create(row_id='5', table='publication', audit_command=self.context)
+        models.AuditLog.objects.create(row_id='5',table='publication', audit_command=self.second_context)
+
+        management.call_command('zotero_import', test=True)
```

Comment:
Better to not invoke Django's management app here and instead extract test data generating logic from zotero_import and invoke it directly. Otherwise we have a lot more going on than what should be a simple unit test. Of course we should be testing the management command as well but that should be a separate test of its own..